contrib/format-pdf/src/main/resources/bootstrap-format-plugins.json

```diff
@@ -0,0 +1,43 @@
+{
+  "storage":{
+    "dfs": {
+      "type": "file",
+      "formats": {
+        "pdf": {
+          "type": "pdf",
+          "extensions": [
+            "pdf"
+          ],
+          "extractHeaders": true,
+          "combinePages": false
+        }
+      }
+    },
+    "cp": {
```

Comment:
Will we provide any PDFs built into Drill?

Reply:
I didn't plan on it.  We can include one in the docs that people can d/l and try out.